Project.toml

```diff
@@ -9,13 +9,17 @@ ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FastGaussQuadrature = "442a2c76-b920-505d-bb47-c5924d526838"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 GPLikelihoods = "6031954c-0455-49d7-b3b9-3e1c99afaf40"
 KLDivergences = "3c9cd921-3d3f-41e2-830c-e020174918cc"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Optim = "429524aa-4258-5aef-a3af-852621145aeb"
+QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
```

Comment:
EP needs some kind of 1D integral for computing the moments. Could switch to FastGaussQuadrature and remove this dependency later.

Reply:
This would be my personal preference, but only because I found it most straightforward to AD through the things that FastGaussQuadrature produced efficiently. Did you manage to do so with QuadGK, or have you not had to differentiate through quadrature?

Reply:
I didn't get to lml approximation with EP yet! so haven't had the chance to find out. But yeah in this use-case (one factor is a Gaussian) Gauss-Hermite quadrature is probably the best solution anyways. And in the medium term it should get moved over to GPLikelihoods.jl.

Reply:
sorted